pkg/extension/workspace/example/runtime/configdir/configdir_test.go

```diff
@@ -32,50 +32,34 @@ import (
 
 func TestNewConfigDir(t *testing.T) {
 	tests := []struct {
-		name             string
-		workspaceDir     string
-		isEnvoyTemplate  func(string) bool
-		requireBootstrap func(t *testing.T, bootstrap *envoybootstrap.Bootstrap)
+		name                 string
+		workspaceDir         string
+		expectValidBootstrap bool
 	}{
 		{
-			name:         "envoy.tmpl.yaml",
-			workspaceDir: "testdata/workspace1",
-			isEnvoyTemplate: func(name string) bool {
-				return name == "envoy.tmpl.yaml" //nolint:goconst
-			},
-			requireBootstrap: expectValidBootstrap,
+			name:                 "envoy.tmpl.yaml",
+			workspaceDir:         "testdata/workspace1",
+			expectValidBootstrap: true,
 		},
 		{
-			name:         "envoy.tmpl.yaml + lds.tmpl.yaml + cds.yaml",
-			workspaceDir: "testdata/workspace2",
-			isEnvoyTemplate: func(name string) bool {
-				return name == "envoy.tmpl.yaml" || name == "lds.tmpl.yaml" || name == "cds.yaml"
-			},
-			requireBootstrap: expectValidBootstrap,
+			name:                 "envoy.tmpl.yaml + lds.tmpl.yaml + cds.yaml",
+			workspaceDir:         "testdata/workspace2",
+			expectValidBootstrap: true,
 		},
 		{
-			name:         "envoy.tmpl.yaml: not a valid YAML",
-			workspaceDir: "testdata/workspace3",
-			isEnvoyTemplate: func(name string) bool {
-				return false
-			},
-			requireBootstrap: expectInvalidBootstrap,
+			name:                 "envoy.tmpl.yaml: not a valid YAML",
+			workspaceDir:         "testdata/workspace3",
+			expectValidBootstrap: false,
 		},
 		{
-			name:         "envoy.tmpl.yaml: invalid paths to `lds` and `cds` files",
-			workspaceDir: "testdata/workspace4",
-			isEnvoyTemplate: func(name string) bool {
-				return name == "envoy.tmpl.yaml"
-			},
-			requireBootstrap: expectValidBootstrap,
+			name:                 "envoy.tmpl.yaml: invalid paths to `lds` and `cds` files",
```

Comment:
we accidentally had also included lds and cds files in the config directory despite the paths being incorrect. These were templated and made yaml parsing fail. The simpler out was to not include these files in the test about invalid paths to them!